config/webhook/service.yaml

```diff
@@ -1,18 +1,13 @@
-
 apiVersion: v1
 kind: Service
 metadata:
   labels:
-    app.kubernetes.io/name: service
-    app.kubernetes.io/instance: webhook-service
-    app.kubernetes.io/component: webhook
     app.kubernetes.io/created-by: opendatahub-operator
     app.kubernetes.io/part-of: opendatahub-operator
     app.kubernetes.io/managed-by: kustomize
   name: webhook-service
-  namespace: system
   annotations:
-    service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-controller-webhook-cert
+    service.beta.openshift.io/serving-cert-secret-name: opendatahub-operator-webhook-cert
```

Comment:
Are these changes related to the update?

And more general: can I take webhook changes as a separate PR? They do not require the update, can go now already and it will be easier then to take the patch downstream.

Reply:
i can leave the changes in bundle/ + config/webhook/ out from this PR
but the one in the controllers/webhook/ will be needed for 0.17.5 to work

Reply:
Yes, but webhook initialization changes do not require updated runtime and can be submitted right now, don't they?

Reply:
if i understand you correctly:
you want to have another PR only for the webhook decoder part in ODH first, and you want to get it merge into downstream without introduce the upgrade runtime in downstream.

Reply:
> if i understand you correctly: you want to have another PR only for the webhook decoder part in ODH first, and you want to get it merge into downstream without introduce the upgrade runtime in downstream.

I want webhook changes in a separate PR, it may be more general with both cleanup in names and manifests (if you want it) and decoder part. To track it for backporting separately from the upgrade.

Reply:
ok, i have no objection on this

Reply:
i have narrow the changes in this PR, explicitly removed some part of webhook.
but due to the fact with a newer version or controller-runtime the old implementation on webhook has been changed, in this case: https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.6/pkg/webhook/admission#InjectDecoderInto is gone.